google/cloud/aiplatform/jobs.py

```diff
@@ -364,6 +365,7 @@ def create(
         accelerator_count: Optional[int] = None,
         starting_replica_count: Optional[int] = None,
         max_replica_count: Optional[int] = None,
+        batch_size: Optional[int] = None,
```

Comment:
This argument should be added to the end of the method signature to avoid breaking change based on order of arguments.